src/shared/view/nunjucks/filters/gauging-station-badge.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const badge = {
+  no_restriction: { status: 'success', text: 'no restriction' },
```

Comment:
I **_think_** we call that `resume` rather than `no_restriction`.

This is because licences may have a status of `resume` or `null` but they'll never have a status that says `no_restriction`.

I might be wrong here so feel free to ignore this